### PR TITLE
Calculations

### PR DESCRIPTION
**Pull request by meronogbai**
- Setup main logic of the app.
- Implemented calculate and operate functions.
- Used big.js.

### CODE REVIEW

File: src/logic/calculate.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import operate from './operate';
+
+export default function calculate(calcData, buttonName) {
+  let { total, next, operation } = calcData;
+  let result;
+  switch (buttonName) {
+    case '+':
+    case '-':
+    case 'X':
+    case '÷':
```
Comment (whiz25) [inline src/logic/calculate.js:9]: - Implement operations for `+/-` and `.` buttons

File: src/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      result = x.times(y);
+      break;
+    case '÷':
+      result = x.div(y);
+      break;
+    case '+/-':
+      result = x.times(y).times(-1);
+      break;
+    default:
+      break;
```
Comment (whiz25) [inline src/logic/operate.js:24]: - Include implementation for the `%` operator. This is very common operator in a calculator.
```javascript
case '%':
    result = y.mod(x)
```

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required :pray:
Job well done. You implemented most of the requested features for this PR. Your commit messages and the README are professional. 
However, you missed implementing some core functionalities of the calculator app. Checkout my comments for some suggestions.


### Martin Afani

File: src/logic/calculate.js (hunk 1 above)
Comment (meronogbai) [reply src/logic/calculate.js:9]: Done

File: src/logic/operate.js (hunk 2 above)
Comment (meronogbai) [reply src/logic/operate.js:24]: Done

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi @meronokbay, congratulations on this project :+1: Great job fixing the comments about the `operate` and `calculate` logic as mentioned in the previous review :fireworks: Awesome
Based on this I hereby approve the project :smile: 

Kindly go ahead and merge the PR branch into the dev branch :+1: Do not forget to submit the project completion form
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**